pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -239,9 +239,12 @@ type CatalogSourceStatus struct {
 	// The last time the CatalogSource image registry has been polled to ensure the image is up-to-date
 	LatestImageRegistryPoll *metav1.Time `json:"latestImageRegistryPoll,omitempty"`
 
-	ConfigMapResource     *ConfigMapResourceReference `json:"configMapReference,omitempty"`
-	RegistryServiceStatus *RegistryServiceStatus      `json:"registryService,omitempty"`
-	GRPCConnectionState   *GRPCConnectionState        `json:"connectionState,omitempty"`
+	// ConfigMapReference (deprecated) is the reference to the ConfigMap containing the catalog source's configuration, when the catalog source is a ConfigMap
+	ConfigMapResource *ConfigMapResourceReference `json:"configMapReference,omitempty"`
+	// RegistryService is the GRPC service that the catalog source is using to serve the catalog
```

Comment:
This comment doesn't actually describe the meaning of this field, it just describes what the field is describing

Reply:
Well, phooey.  Thought I'd fixed that chatgpt'd fluff before I opened it.  Updated now.